coral/src/app/features/requests/components/acls/AclRequests.test.tsx

```diff
@@ -8,14 +8,20 @@ import userEvent from "@testing-library/user-event";
 import { AclRequests } from "src/app/features/requests/components/acls/AclRequests";
 import { getAclRequests } from "src/domain/acl/acl-api";
 import transformAclRequestApiResponse from "src/domain/acl/acl-transformer";
+import { getEnvironments } from "src/domain/environment";
+import { createEnvironment } from "src/domain/environment/environment-test-helper";
 import { mockIntersectionObserver } from "src/services/test-utils/mock-intersection-observer";
 import { customRender } from "src/services/test-utils/render-with-wrappers";
 
 jest.mock("src/domain/acl/acl-api.ts");
+jest.mock("src/domain/environment/environment-api.ts");
 
 const mockGetAclRequests = getAclRequests as jest.MockedFunction<
   typeof getAclRequests
 >;
+const mockGetEnvironments = getEnvironments as jest.MockedFunction<
```

Comment:
You need to add this mock to the tests to avoid react query running into errors because it get's no value from the api call. You can see the errors if you run the single test (they get swallowed in the whole test suite). 

Adding a `mockGetEnvironments.mockResolvedValue([])` in the first beforeEach loop that covers all test would be enough to solve this.

Reply:
I did it like this, seems like it addresses the issue :o [5e12878](https://github.com/aiven/klaw/pull/852/commits/5e128781a12b69c23288e1421328865cfdf1a11f)